setup-knative-eventing/action.yaml

```diff
@@ -9,9 +9,9 @@ description: |
 inputs:
   version:
     description: |
-      The version of Knative to install, e.g. 1.5.0
+      The version of Knative to install, e.g. 1.8.0
     required: false
-    default: 1.5.0
+    default: 1.8.0
```

Comment:
I think this broke @vaikas in scaffolding?

Reply:
once https://github.com/sigstore/scaffolding/pull/431 lands, we can bump the version without concern of breaking scaffold

Reply:
once https://github.com/sigstore/scaffolding/pull/431 lands, scaffold will be pinned to 1.7.x, and using latest.